src/mode/buffered_graphics.rs

```diff
@@ -69,9 +69,14 @@ where
     SIZE: DisplaySize,
 {
     /// Clear the display buffer. You need to call `disp.flush()` for any effect on the screen
+    #[cfg(not(feature = "graphics"))]
     pub fn clear(&mut self) {
+        self.clear_impl(false);
+    }
+
+    fn clear_impl(&mut self, value: bool) {
         for b in self.mode.buffer.as_mut() {
-            *b = 0;
+            *b = value as u8;
         }
```

Comment:
Isn't this the same as `self.mode.buffer.fill(value as u8)`? Maybe that's faster if the compiler didn't already optimize the original code into the same thing.

Reply:
You're right, this is some sort of fill. On the other hand, this doesn't matter in term of performance as much as not using `fill_solid` is.